pkgs/tools/security/exploitdb/default.nix

```
@@ -0,0 +1,24 @@
+{stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "exploitdb";
+  version = "2021-05-22";
+  src = fetchFromGitHub {
```

Better readability for humans :wink: 

```suggestion
  version = "2021-05-22";

  src = fetchFromGitHub {
```